src/nwb_datajoint/common/common_spikesorting.py

```diff
@@ -459,6 +459,36 @@ def insert_default(self):
             sorter_params = ss.get_default_params(sorter)
             self.insert1([sorter, 'default', sorter_params], skip_duplicates=True)
 
+    def insert_franklab_defaults(self):
```

Comment:
can you just make this part of `insert_default`? that way user doesn't need to call both.

Reply:
done